src/classes/filter/EngineLevelFilter.as

```diff
@@ -166,6 +167,9 @@ package classes.filter
                 case FILTER_SONG_FLAGS:
                     return compareSongFlag(songInfo, userData.getLevelRank(songInfo), input_number);
 
+                case FILTER_AAA_EQUIV:
+                    return greaterThan(songInfo.difficulty, userData.skill_rating_songs[userData.skill_rating_top_count - 1].equiv) && userData.getLevelRank(songInfo).rawscore < songInfo.score_raw;
```

Comment:
What happens in the event `skill_rating_songs` is less then 50 songs, such as a new user?
At a glance it would seem like it would produce a out-of-bounds error.